frontend/src/apiAccess/apis.js

```diff
@@ -353,12 +353,7 @@ async function setProjectLinkSubmission({ cardId, linkSubmission }) {
   return { response, responseData };
 }
 
-async function activityLogDayCountsPage({
-  eventTypeName,
-  actorUser,
-  effectedUser,
-  page,
-}) {
+async function activityLogDayCountsPage({ eventTypeName, actorUser, page }) {
```

Comment:
what about the effected user? If I receive a review then I would want to know about it. Can you please put the filter back.

Can you also please add the rest of the filters? Here is a list from the backend:

```
filterset_fields = [
        "event_type",
        "actor_user",
        "effected_user",
        "object_1_content_type",
        "object_1_id",
        "object_2_content_type",
        "object_2_id",
    ]
```